components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -106,6 +123,28 @@ + (nonnull instancetype)sampleWithTitle:(nonnull NSString *)title
   return sample;
 }
 
+- (void)addMDCButtonWithFrame:(CGRect)frame
+                 buttonScheme:(id<MDCButtonScheming>)buttonScheme
+                        title:(NSString *)title
+                  actionBlock:(MDCButtonActionBlock)actionBlock {
+  MDCButton *button = [[MDCButton alloc] initWithFrame:CGRectZero];
+  [button setTitle:title forState:UIControlStateNormal];
+  [MDCContainedButtonThemer applyScheme:buttonScheme toButton:button];
+  [self.view addSubview:button];
+  self.button = button;
+  self.buttonFrame = frame;
```

Comment:
Please consider calling CGRectStandardize before assigning the property.

Reply:
Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.